packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -39,6 +39,15 @@ The following operators and functionalities offered by SpEL aren't supported in
 
 ## Reference attributes
 
+### Application entitlements
+
+When you create an Okta expression, you can reference entitlements within the `appuser` context.
```

Comment:
Do we need a callout that this is only supported for federated claims? I.e. you cannot utilize them in authentication policies yet